frontend/package.json

```diff
@@ -38,17 +41,22 @@
     "yup": "^0.28.2"
   },
   "devDependencies": {
+    "@rushstack/eslint-patch": "^1.2.0",
```

Comment:
should add context: https://rushstack.io/ and https://github.com/microsoft/rushstack

Reply:
also good catch, I'm 99% sure this isn't actually being used